quickwit-indexing/src/actors/merge_split_downloader.rs

```diff
@@ -25,18 +25,17 @@ use quickwit_metastore::SplitMetadata;
 use tantivy::Directory;
 use tracing::{info, info_span, warn, Span};
 
-use crate::actors::MergeExecutor;
 use crate::merge_policy::MergeOperation;
 use crate::models::{MergeScratch, ScratchDirectory};
 use crate::split_store::IndexingSplitStore;
 
-pub struct MergeSplitDownloader {
+pub struct MergeSplitDownloader<A: Actor> {
```

Comment:
what is the generics about?

Reply:
Good point. 

I changed the name "A" for `ExecutorActor` as what I want is to reuse the `MergeSplitDownloader` but send the download split to the `DeleteTaskExecutor`.